core/rawdb/prunedfreezer.go

```diff
@@ -300,5 +300,5 @@ func (f *prunedfreezer) AncientRange(kind string, start, count, maxBytes uint64)
 }
 
 func (f *prunedfreezer) ModifyAncients(func(ethdb.AncientWriteOp) error) (int64, error) {
-	return 0, errNotSupported
+	return 0, nil
```

Comment:
`prunedfreezer` should not support `ModifyAncients()`, since it does not contain ancient data. So unreasonable to modify ancient.

Reply:
thanks, seems this PR is unnecessary, I'll close it right now